packages/components/src/DialogActions/styles.ts

```diff
@@ -4,7 +4,11 @@ import { Grid, type GridProps } from '../Grid';
 import { styled } from '../styles';
 
 export const StyledDialogActions = styled(DialogActions)`
-  padding-top: 0;
+  padding-top: ${({ theme }) => theme.spacing(0)};
```

Comment:
```suggestion
  padding-top: 0;
```

Обращение к теме не нужно